elements/pfe-navigation/demo/index_react.html

```diff
@@ -175,31 +201,78 @@ <h3>
+                      <a href="#">Red Hat JBoss Enterprise Application Platform</a>
+                    </li>
+                    <li>
+                      <a href="#">Red Hat OpenStack Platform</a>
+                    </li>
+                    <li class="pfe-navigation__sub-nav-link--separator">
+                      <a href="#">Red Hat Virtualization</a>
+                    </li>
+                    <li>
+                      <a href="#">Red Hat Virtualization Example</a>
+                    </li>
+                  </ul>
+                </div>
               </li>
-              <li className="pfe-navigation__menu-item">
-                <a href="#" className="pfe-navigation__menu-link">
+              <li class="pfe-navigation__menu-item">
+                <a href="#" class="pfe-navigation__menu-link">
                   Resources
                 </a>
               </li>
-              <li className="pfe-navigation__menu-item">
-                <a href="#" className="pfe-navigation__menu-link">
-                  Red Hat &amp; Open Source
+              <li class="pfe-navigation__menu-item">
+                <a href="#" class="pfe-navigation__menu-link">
+                  Red Hat & Open Source
+                </a>
+              </li>
+              <li class="pfe-navigation__menu-item">
+                <a href="#" class="pfe-navigation__menu-link">
+                  Really Long Menu Title
                 </a>
               </li>
             </ul>
           </nav>
-          <div slot="pfe-navigation--custom-links" className="pfe-navigation__custom-links" id="pfe-navigation__custom-links">
-            <a href="#" className="">Custom Link</a>
-          </div>
-          <div slot="pfe-navigation--search" className="pfe-navigation__search">
+
+          <ul class="pfe-navigation__fallback-links">
+            <li>
+              <a href="#">Search</a>
+            </li>
+            <li>
+              <a href="#">Custom Link</a>
+            </li>
+            <li>
+              <a href="#">Log in</a>
+            </li>
+          </ul>
+
+          <li slot="pfe-navigation--custom-links">
+            <a href="#">
+              <pfe-icon icon="web-icon-globe" pfe-size="md" aria-hidden="true"></pfe-icon>
+              Custom Link
+            </a>
+          </li>
+          <li slot="pfe-navigation--custom-links">
+            <button slot="pfe-navigation--custom-links">
+              <pfe-icon icon="web-cart" pfe-size="md" aria-hidden="true"></pfe-icon>
+              Cart
+            </button>
+          </li>
+
+          <div slot="pfe-navigation--search" class="pfe-navigation__search pfe-navigation__search--default-styles">
             <form>
-              <label>Search</label>
-              <input type="text" placeholder="Search" />
-              <button>Search</button>
+              <label for="pfe-navigation__search-label1" class="sr-only">Search the Red Hat Customer Portal</label>
```

Comment:
I believe this will need to be `htmlFor` instead of `for`

Reply:
See comment above ^